stravalib/client.py

```diff
@@ -85,15 +90,22 @@ def __init__(
 
     @property
     def access_token(self):
-        """
-        The currently configured authorization token.
-        """
+        """The currently configured authorization token."""
         return self.protocol.access_token
 
     @access_token.setter
     def access_token(self, v):
-        """
-        Set the currently configured authorization token.
+        """Set the currently configured authorization token.
```

Comment:
@jsamoocha one other thought i had ... 

there are some instances of a variable like this - i suspect v == value. i thought about making the variables more expressive to changing v -> token_value or token. i suppose if some people have hard coded v then it could break things in multi variable input methods .

Reply:
> i suppose if some people have hard coded v then it could break things in multi variable input methods .

That's a valid concern, but in this case, we're dealing with a single positional argument of a property setter. This will be called as `client.access_token = SOME_TOKEN`, so the risk of breaking stuff is minimal. It makes sense to use a more descriptive variable name here.

Reply:
oh that's right. i didn't pay attention to the decorator i was just in fix everything mode. ok i'll update.